packages/manager/src/features/LoadBalancers/LoadBalancerLanding/RegionsCell.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+import { useRegionsQuery } from 'src/queries/regions';
+
+interface Props {
+  region: string;
+}
+
+export const RegionsCell = (props: Props) => {
+  const { region } = props;
+  const { data: regions } = useRegionsQuery();
+
+  const actualRegion = regions?.find((r) => r.id === region);
+
+  return (
+    <div style={{ alignItems: 'center', whiteSpace: 'nowrap' }}>
+      {actualRegion?.label ? `${actualRegion.label} (${region})` : region}
+    </div>
```

Comment:
`alignItems` isn't doing anything here because div is not flex by default

Maybe we could make this 

```suggestion
    <Typography noWrap>
      {actualRegion?.label ? `${actualRegion.label} (${region})` : region}
    </Typography>
```